graphdatascience/tests/integration/conftest.py

```diff
@@ -71,6 +75,12 @@ def gds_without_arrow() -> GraphDataScience:
 
 
 def pytest_collection_modifyitems(config: Any, items: Any) -> None:
+    if config.getoption("--target-aura"):
```

Comment:
maybe .. since we call the other one `--include-enterprise` we can do `--exclude-aura`?

Reply:
it's a nit-pick so feel free to ignore. The other options are also kind of randomly named.

Reply:
Not sure I think the "exclude-aura" naming is great if it's an option for actually testing against aura. Maybe something like "exclude-self-managed-only" would be more logical, but too complicated I think. I will keep as is for now. We can rename later if we want